engine/api/api.go

```diff
@@ -576,6 +579,9 @@ func (a *API) Serve(ctx context.Context) error {
 				BucketName:          a.Config.Artifact.AWSS3.BucketName,
 				Region:              a.Config.Artifact.AWSS3.Region,
 				SessionToken:        a.Config.Artifact.AWSS3.SessionToken,
+				Endpoint:            a.Config.Artifact.AWSS3.Endpoint,
+				DisableSSL:          a.Config.Artifact.AWSS3.DisableSSL,
+				S3ForcePathStyle:    a.Config.Artifact.AWSS3.S3ForcePathStyle,
```

Comment:
maybe rename this with `ForcePathStyle` could be better to not repeat the `S3` part